internal/service/credentials/resource_credential_issuer_profile.go

```diff
@@ -345,33 +351,11 @@ func (p *CredentialIssuerProfileResourceModel) expand() (*credentials.Credential
 
 	data.SetApplicationInstance(*applicationInstanceId)
 
-	if !p.CreatedAt.IsNull() && !p.CreatedAt.IsUnknown() {
-		createdAt, err := time.Parse(time.RFC3339, p.CreatedAt.ValueString())
-		if err != nil {
-			diags.AddWarning(
-				"Unexpected Value",
-				fmt.Sprintf("Unexpected createdAt value: %s.  Please report this to the provider maintainers.", err.Error()),
-			)
-		}
-		data.SetCreatedAt(createdAt)
-	}
-
-	if !p.UpdatedAt.IsNull() && !p.UpdatedAt.IsUnknown() {
-		updatedAt, err := time.Parse(time.RFC3339, p.UpdatedAt.ValueString())
-		if err != nil {
-			diags.AddWarning(
-				"Unexpected Value",
-				fmt.Sprintf("Unexpected updatedAt value: %s.  Please report this to the provider maintainers.", err.Error()),
-			)
-		}
-		data.SetUpdatedAt(updatedAt)
-
-		if data == nil {
-			diags.AddWarning(
-				"Unexpected Value",
-				"Credential Issuer Profile object was unexpectedly null on expansion.  Please report this to the provider maintainers.",
-			)
-		}
+	if data == nil {
+		diags.AddWarning(
```

Comment:
This check probably should have been picked up by the linter, because line 352 expects an object already exists and is not `nil`, this `if` statement will never run.  It can be safely removed

Reply:
Corrected.